benchexec/tools/ultimate.py

```diff
@@ -357,7 +354,7 @@ def _determine_result_without_propertyfile(
         ltl_true_string = "Buchi Automizer proved that the LTL property"
         overflow_false_string = "overflow possible"
 
-        for line in output:
+        for line in run.output:
             if line.find(unsupported_syntax_errorstring) != -1:
```

Comment:
Just noticed this: Looking for strings in Python is most easily expressed as `if needle in haystack:`